<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="DialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="nameEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="roastEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>степень обжарки</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>тип кофе (молотый/в зернах)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="descEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>описание</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>210</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;цена&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>цена</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="volumeEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>250</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;цена&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>количество</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>310</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
